api/PclusterApiHandler.py

```diff
@@ -685,9 +690,59 @@ def _get_params(_request):
     params.pop("path")
     return params
 
+def check_tags():
+    try:
+        cost_explorer = boto3.client('ce',)
+        inactive_tags = cost_explorer.list_cost_allocation_tags(Status='Inactive',TagKeys=PCLUSTER_COST_TAGS)
+        return {'TagStatus': not (len(inactive_tags['CostAllocationTags']) > 0)}
+    except Exception as e:
+        return {"message": str(e)}, 500
 
-# Proxy
+def activate_tags():
+    try:
+        cost_explorer = boto3.client('ce')
+        newlist = [{'TagKey': tag, 'Status': 'Active'} for tag in PCLUSTER_COST_TAGS]
+        cost_explorer.update_cost_allocation_tags_status(CostAllocationTagsStatus=newlist)
+        return "Activated Tags"
```

Comment:
I'd suggest returning the same shape here as you do when there's an error, so that the parsing is consistent on the frontend.